lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -517,8 +521,13 @@ def rescore_problem_module_state(xmodule_instance_args, module_descriptor, stude
             msg = "Specified problem does not support rescoring."
             raise UpdateProblemModuleStateError(msg)
 
+        # set the tracking info before this call, because
+        # it makes downstream calls that create events
+        action_id = create_new_user_action_id()
+        set_user_action_type('edx.grades.problem.rescored')
```

Comment:
Nit: this string is used 3 times in this method.  Can we create a reusable constant for it?